__tests__/app.test.js

```diff
@@ -86,4 +71,26 @@ describe('GET /api/articles/:article_id', () => {
             expect(body.msg).toBe('Invalid article ID');
         })
     })
+})
+
+
+describe('GET /api/articles', () => {
+    test('responds with an array of article objects, each with the correct properties', () => {
+        return request(app).get('/api/articles').expect(200).then(({ body }) => {
+            const testArticle = {
+                author: expect.any(String),
+                title: expect.any(String),
+                article_id: expect.any(Number),
+                topic: expect.any(String),
+                created_at: expect.any(String),
+                votes: expect.any(Number),
+                article_img_url: expect.any(String),
+                comment_count: expect.any(String)
+            }
+            expect(body.articles).toHaveLength(13);
+            body.articles.forEach((article) => {
+                expect(article).toMatchObject(testArticle);
+            })
+        })
```

Comment:
Solid test 🧪 , great that you've checked the length of the returned array so you're not risking a false positive if there's an empty array 👍 
Could do with also checking that there is no body property on the returned articles objects

Reply:
Aweesome thanks, will add that body test before i merge it. Cheers